src/analyzer.ts

```diff
@@ -117,16 +117,29 @@ export class Analyzer {
   }
 
   /**
+   * Returns `true` if the provided resolved URL can be loaded.
```

Comment:
`Analyzer implements UrlLoader` would say as much?  But that feels so wrong.  I'll just put it in the comments for now.